Packs/FeedTAXII/Integrations/FeedTAXII2/FeedTAXII2.py

```diff
@@ -48,6 +48,48 @@ def module_test_command(client, limit, fetch_full_feed):
         return_error("Could not connect to server")
 
 
+def filter_indicators(indicators: list, last_run: dict) -> list:
+    """
+    Filter the indicators returned from the taxii server, by taking out indicators that we're ingested in the previous fetch
+    call and their modified date was not updated,
+    """
+    last_indicators = last_run.get("latest_indicators")  # indicators from prev fetch
+    new_indicators = []
+    if not last_indicators:    # first fetch
+        last_run["latest_indicators"] = [{obj.get('rawJSON', {}).get("id"): obj.get('rawJSON', {}).get("modified")}
+                                         if obj.get("value") != "$$DummyIndicator$$" else obj
+                                         for obj in indicators]
+        return indicators
+    for indicator in indicators:
+        indicator_id = indicator.get("rawJSON", {}).get('id', "")
+
+        # check if the indicator is stored in latest_indicators
+        saved_indicator = list(filter(lambda ind: indicator_id in ind.keys(), last_indicators))
+
+        # if the indicator is stored in latest_indicators -> check if it was modified
+        if saved_indicator:
+            modified_date = saved_indicator[0].get(indicator_id)
+
+            # the indicator is stored in latest_indicators, but got modified -> add to new_indicators
+            if indicator.get("rawJSON", {}).get("modified", "") > modified_date:
+                new_indicators.append(indicator)
+            else:
+                demisto.debug(f"indicator {indicator_id} was already ingested in the previous fetch...skipping.")
```

Comment:
instead of logging every time please collect them in a list and in the end of the loop plcheck if it is not empty.
if this is not empty, do 2 log lines:
info: the amount of filtered indicators.
debug: the list itself if the ids.

this way it will not spam the logs but still be useful for debugging.